application/frontend/config/main.php

```diff
@@ -1,32 +1,43 @@
 <?php
 
+use Sil\Idp\IdSync\common\components\ApiConsumer;
```

Comment:
I think we're tending towards a namespace of `Sil\Idp\_RepoNamespace_\...` for code related to this IdP work, so it probably wouldn't hurt. It's probably not worth a ton of time to go back and refactor stuff, though, if any of it doesn't cooperate.